doc/build-unix.md

```diff
@@ -1,30 +1,9 @@
-UNIX BUILD NOTES
+UNIX Build Instructions and Notes
 ====================
-Some notes on how to build Bitcoin Core in Unix.
+Some notes on how to build Elements in Unix.
 
 (for OpenBSD specific instructions, see [build-openbsd.md](build-openbsd.md))
 
-Note
```

Comment:
I'm not sure this movement helps. But if it does, then we should try to upstream it to bitcoin.